tests/integration/records/views/test_views_jobs.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+def test_jobs_application_json_get(api_client, db, create_record_factory):
```

Comment:
this tests are quite useless, since we're not testing the actual API response for each "Content-Type"

Reply:
Still this tests are moved from test_views.py This are not tests created by me. You never told me to also redo this tests, you only requested to move them.